src/lib/network_pool/transaction_pool.ml

```diff
@@ -1113,6 +1127,17 @@ struct
                               (* We can't punish peers for this, since these
                                    are our specific preferences.
                                 *)
+                              if is_sender_local then
```

Comment:
Do we still need these additional logs?

Reply:
no ill clean them up before merging

Reply:
well as for these logs, should those other events be emitting `Rejecting_command_for_reason` structured events?  i added them because i thought maybe they should be, but it's true that that isn't the issue after all

Reply:
I guess we do actually. I thought these logs were handled at the `network_pool_base.ml` layer, but looks like that's not the case. Leave them in place for now so that we will see them as errors in the int test framework, should they ever happen.